avocado_vt/options.py

```diff
@@ -474,7 +490,22 @@ def _process_options(self):
         self.cartesian_parser = cartesian_config.Parser(debug=False)
 
         if self.options.vt_config:
-            cfg = os.path.abspath(self.options.vt_config)
+            config_files = ['tests.cfg', 'subtests.cfg', 'guest-os.cfg',
+                            'machines.cfg', 'guest-hw.cfg', 'virtio-win.cfg',
+                            'guest-hw.cfg', 'cdkeys.cfg', 'base.cfg',
+                            'tests-shared.cfg', 'drivers.cfg']
+            local_cfg = os.path.abspath(self.options.vt_config)
+            local_cfg_filename = local_cfg.split('/')[-1]
+            if local_cfg_filename not in config_files:
+                cfg = data_dir.get_backend_cfg_path(self.options.vt_type,
+                                                    local_cfg_filename)
+                # Skip if cfg path is happended to local config path given
+                if local_cfg != cfg:
+                    copyfile(local_cfg, cfg)
+            else:
+                raise ValueError("Invalid test config file name: %s\n "
```

Comment:
@xutian Oh, sure, wanted add it in my next version based on our discussion whether to go with way or not, 
based on our conclusion will send with nitpick, will remove it for now.
Thanks a lot :-)